### DISCUSSION TITLE
Generating thumbnail preview in Explorer blocks file selection

### DISCUSSION THREAD

**Discussion by gaga9999** [sumatrapdfreader/sumatrapdf#2333] [Q&A]
Hy!
While in Windows Explorer, when i select a PDF it renders the preview. While that is done Windows blocks the selection of another file. I never had this in previous windows versions, is this sumatra specific and can it be fixed? (Win10). Thanks

**Comment by GitHubRulesOK** [comment]
Generally should not be a significant delay, however on occasions I have seen similar. My guess is either SumatraPDF is choking on the type of PDF (attempting to render first page) or the AV watching over me is analysing the pdf for viral activity, eventhough SumatraPDF does not run the javascript content itself OR there is a nother process involved.

So is it just larger files or is it a time block thats equal for a mix of different types/sizes?

In my case it only seems to be for short periods and clears until the next time which could be days later or in 10 minutes if windows is struggling.

**Comment by kjk** [comment]
I don't think there's anything I can do in Sumatra to fix that. Seems like a Windows / Explorer issue.

**Comment by gaga9999** [comment]
ok thanks. @githubrules its always the same delay, and its kind of long

**Reply by GitHubRulesOK** [reply]
Sounds like some app is looking at the previewer touching the file and then do its own thing so consider AntiVirus, 3rd party File explorers or searchers (like "everything") or any file related enhancements that did not come with vanilla windows.

Or even a second previewer
